src/components/molecules/ChatsList/ChatsList.tsx

```diff
@@ -105,18 +102,19 @@ const ChatsList: React.FunctionComponent = () => {
     [privateChats, selectedUser]
   );
 
-  const chatContext = useContext(ChatContext);
   const submitMessage = useCallback(
     async (data: { messageToTheBand: string }) => {
-      chatContext &&
-        user &&
-        chatContext.sendPrivateChat(
-          user.uid,
-          selectedUser!.id,
-          data.messageToTheBand
+      user &&
+        dispatch(
+          sendPrivateChat({
+            type: SEND_PRIVATE_CHAT,
+            from: user.uid,
+            to: selectedUser!.id,
+            text: data.messageToTheBand,
+          })
```

Comment:
We shouldn't manually construct an action in a component like this. This is what action creator functions are used for.

https://redux-toolkit.js.org/tutorials/basic-tutorial#introducing-createaction